### ISSUE TITLE
[BUG] KYC asks for camera, but after taking the picture nothing is uploaded

### ISSUE THREAD

**Issue by georgemac70cr8** [asgiom/cr8-brm-mobile-issues#15]
**NOTHING HAPPENS WHEN CHOOSING TO TAKE A PHOTO OR UPLOAD FILE**

I now have 2 choices for either taking a photo or using file, but once I select file or take photo it doesn’t accept / process it.

[https://vimeo.com/user121655593/review/449227947/f79b49f201]

**To Reproduce**
1. Go to 'buy/sell'
2. See attached video - apologies for my look!

**Expected behavior**
Obviously, what should happen is that at uploads the file (and gives you appropriate message if not image file), or lets you submit the photo you have just taken.

**Screenshots**
video attached

**Smartphone (please complete the following information):**
HUWAEI P20 PRO 
Android OS version 10
SAMSUNG GALAXY S9
Andorid OS version ?
So probably more phones too

**Comment by t3nshi** [comment]
When we asked on the issue template the OS version is because this information becomes very relevant sometimes for example in here. Can you provide OS version of each phone so we can narrow down the issue? thanks in advance.

**Comment by georgemac70cr8** [comment]
I gave the os a while back i will give again.

**Comment by t3nshi** [comment]
Thanks for your quick response, I removed the images just because I wouldn't feel comfortable if my IMEI was hanging around in some web... I just needed the OS version, I already updated on the issue info... if you gave it before in some other issue try to remember so you include it in a new issue, it will be easier for you to remember your os version of your phone to let me know than for me to remember each os version of the dozen of phones of the testers on the issues. In the picture was android 10 for the P20, and the picture on the s9 didn't show anything at all except the IMEI... so when you figure it out please let me know. thanks!

**Comment by georgemac70cr8** [comment]
so you need more detail for the samsung galaxy phone?

**Comment by t3nshi** [comment]
this seems to be already mentioned on #8

**Comment by t3nshi** [comment]
When we have an open issue about the same thing is worthwhile to continue the discussion there to keep other parties informed instead of opening a new issue, unless is different and unrelated.

**Comment by t3nshi** [comment]
> so you need more detail for the samsung galaxy phone?

Please just tell me OS version, so  we can confirm if this only happens on phones with android 10.

**Comment by georgemac70cr8** [comment]
it's mentioned but it isn't quite the same problem. before there was no option to either use camera or use a file. now the problem is differnt because we have the option to choose but now there is no repsonse afterwards.

**Comment by georgemac70cr8** [comment]
i'm happy for you to merge these 2 issues into 1

**Comment by t3nshi** [comment]
> it's mentioned but it isn't quite the same problem. before there was no option to either use camera or use a file. now the problem is differnt because we have the option to choose but now there is no repsonse afterwards.

You are right... We are now following this since this has full details on the issue description

**Comment by georgemac70cr8** [comment]
the samsung galaxy s9 is android 10 too
![photo_2020-08-19 15 52 47](https://user-images.githubusercontent.com/69824025/90643716-2844dd80-e234-11ea-9971-f4bdd28f0428.jpeg)

**Comment by moatazkhalifa** [comment]
Same issue on my end.

Samsung S9+ Android 10

**Comment by t3nshi** [comment]
We finally got a device running android 10 and now I can see some useful debugging info... I will let you guys know as soon as I crack it!

**Comment by t3nshi** [comment]
Have you got the chance to check this out on the latest version?

**Comment by georgemac70cr8** [comment]
i had already checkedwith latest version but i uninstalled and reinstalled, cleared cache etc... no change still no camera opens. I can choose camera option but the camera doesn't open.

**Comment by t3nshi** [comment]
I've been testing with new tools for HUAWEI, but it is a slow process. In the mean time, I will appreciate that we could narrow the devices that still present the issue and. check right away if there were some devices that got sorted out. Let me know please.

**Comment by georgemac70cr8** [comment]
I can now upload image files and can KYC as a result of uploads. But the camera still doesn't function when i choose camera.

**Comment by georgemac70cr8** [comment]
I may be wrong but I think 0.50 and 0.5.1 had removed the "camera" and "files" option (at least on my phone). This latest version 0.5.2 now reverts back to that on my phone (huawei p20 pro, android 10, emui 10). however, the same problem that the camera functionality doesn't start when you choose "camera"

![Screenshot_20200910_083735_com opibis cr8 mobile](https://user-images.githubusercontent.com/69824025/92690995-302f0380-f342-11ea-86be-73663ab65de2.jpg)

But this might be progress.
